ngraph/frontend/onnx_import/src/op/loop.cpp

```diff
@@ -70,7 +70,14 @@ namespace ngraph
                     const OutputVector loop_carried_dependencies{std::next(ng_inputs.begin(), 2),
                                                                  ng_inputs.end()};
 
-                    const Subgraph& body_graph{node.get_attribute_value<Subgraph>("body")};
+                    std::map<std::size_t, element::Type_t> subgraph_inputs_types_map;
+                    for (std::size_t i = 2; i < ng_inputs.size(); i++)
```

Comment:
you can iterate over `loop_carried_dependencies` here

Reply:
Done